kitchen.dokken.yml

```
@@ -2,66 +2,55 @@
 driver:
   name: dokken
   privileged: true
-  env: [CHEF_LICENSE=accept]
+  chef_version: <%= ENV['CHEF_VERSION'] || 'latest' %>
 
 transport:
   name: dokken
 
 provisioner:
   name: dokken
-  product_name: chef
-  product_version: <%= ENV['CHEF_VERSION'] || 'latest' %>
-  install_strategy: once
 
 platforms:
   - name: amazonlinux-2
     driver:
       image: dokken/amazonlinux-2
       pid_one_command: /usr/lib/systemd/systemd
-
-  - name: debian-8
+  - name: debian-9
     driver:
-      image: dokken/debian-8
+      image: dokken/debian-9
       pid_one_command: /bin/systemd
       intermediate_instructions:
         - RUN /usr/bin/apt-get update
-
-  - name: debian-9
+  - name: debian-10
     driver:
-      image: dokken/debian-9
+      image: dokken/debian-10
       pid_one_command: /bin/systemd
       intermediate_instructions:
         - RUN /usr/bin/apt-get update
-
   - name: centos-7
     driver:
       image: dokken/centos-7
       pid_one_command: /usr/lib/systemd/systemd
-
   - name: centos-8
     driver:
       image: dokken/centos-8
       pid_one_command: /usr/lib/systemd/systemd
-
-  - name: fedora-31
+  - name: fedora-32
```

I recommend you use the following so we don't have to keep updating it on every release.
```suggestion
  - name: fedora-latest
```